customerio-reactnative-richpush.podspec

```diff
@@ -15,17 +15,23 @@ Pod::Spec.new do |s|
   s.platforms    = { :ios => "13.0" }
   s.source       = { :git => "https://github.com/customerio/customerio-ios.git", :tag => "#{s.version}" }
 
-  s.source_files = "ios/**/*.{h,m,mm,swift}"
+  s.source_files = "ios/wrappers/**/*.{h,m,mm,swift}"
 
   # s.dependency "X", "X"
   # Careful when declaring dependencies here. All dependencies will be included in the App Extension target in Xcode, not the host iOS app.   
 
   # Subspecs allow customers to choose between multiple options of what type of version of this rich push package they would like to install. 
   s.subspec 'apn' do |ss|
     ss.dependency "CustomerIO/MessagingPushAPN", package["cioNativeiOSSdkVersion"]
-  end
+    ss.resource_bundles = {
```

Comment:
I tried keeping it in parent `podspec` to avoid duplication, but it [didn't work](https://github.com/customerio/customerio-reactnative/pull/345/commits/62a1553fb1c948c6d0efd87ceb21088ece1e39d4) for some reason. So added it to each `subspec`.